### DISCUSSION TITLE
i have an idea

### DISCUSSION THREAD

**Discussion by CMS-CORP** [JoomCoder-com/JoomCCK#251] [Ideas]
hello as soon as joomcck is finished , i would like to buy  plenty of licences to sell to compagnies like new business solution with joomla in europa. is it possible or you can create special licence for that

**Comment by Giorgi625** [comment]
What do you mean in Finished? because there is Stable version already released which can be used on live projects.
There are new features and bug fixed coming which is usual for every extension. There is not good extension which stops releasing updates, So I think its safe to do it now and it will even help Developer to spend more time on improving specifically JoomCCK.

**Reply by CMS-CORP** [reply]
yes sure i meant , the work is amazing, very very amazing and we hope few things will be implemented . this week i'm going to buy the first licence for our team and we are going to promote joomcck in few week. we have some pro customers in crypto, bank and schools

**Reply by Giorgi625** [reply]
You can use it in many different scenarios, so you will not be disappointed.
